src/I18n/Translator.php

```diff
@@ -157,6 +157,11 @@ public function translate(string $key, array $tokensValues = []): string
 
         if ($message === '') {
             $message = $key;
+
+            //If singular haven't been translated, fallback to the key.
+            if (isset($tokensValues['_singular']) && $tokensValues['_count'] <= 1) {
```

Comment:
```suggestion
            if (isset($tokensValues['_singular']) && $tokensValues['_count'] === 1) {
```

_0 years_ vs _0 year_ (assuming source language is English)